Code/JavaWrappers/ROMol.i

```diff
@@ -289,6 +290,17 @@ void setPreferCoordGen(bool);
     return RDKit::replaceCore(*($self), coreQuery, replaceDummies, labelByIndex);
   };
 
+  /* Methods from SubstructUtils.h */
+  const RDKit::MatchVectType &getMostSubstitutedCoreMatch(const RDKit::ROMol& core,
+    const std::vector<RDKit::MatchVectType>& matches) {
+    return RDKit::getMostSubstitutedCoreMatch(*($self), core, matches);
```

Comment:
I love it when I run across stuff in the public API that I have completely forgotten about. 🤷